bitmark-references/bitmark-cli/bitmark-cli.md

```diff
@@ -0,0 +1,729 @@
+# bitmark-cli Command Reference
+
+## Basic Command Structure
+
+~~~
+bitmark-cli [global-options] command [command-options]
+~~~
+
+## Global Options
+
+Options are listed as long form with any single character abbreviation.
+
+`--verbose|-v`
+
+: This allows the CLI to output additional information for each
+command.  Most commands will output the JSON request and response sent
+to the bitmarkd.
+
+`--network|-n  [bitmark|testing|local]`
+
+: Determines which network the command will be sent to from the following possible values:
+* `bitmark` the live network which uses live BTC or LTC to pay for the transactions.
+* `testing` a network for testing newly developed programs, it uses testnet coins to pay for transactions.
+* `local` a special case for running a regression test network on the loopback interface.
+
+`--connection|-c NUMBER`
+
+: Optionally allows for one of the other connections listed in the
+configuration file to be used instead of defaulting to the first one.
+
+`--identity|-i NAME`
+
+: Optionally override the default identity to allow running commands
```

Comment:
Same comment as for the `connection`. Would it be better if we explain what an `identity` is?